packages/client-node/src/result_set.ts

```diff
@@ -35,34 +38,64 @@ export class ResultSet implements BaseResultSet<Stream.Readable> {
 
     validateStreamFormat(this.format)
 
-    let decodedChunk = ''
+    let data: Buffer
+    let incompleteRow: Buffer | undefined
     const toRows = new Transform({
```

Comment:
Tests WIP. I need to generate a dataset, something with walls of text inside, that will be failing on the current main branch.